invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/collections/grid.variables

```diff
@@ -1,4 +1,4 @@
 /***********************************************
          Invenio App RDM Grid Variables
 ***********************************************/
-@gridLightGreyBackgroundColor: #F8F8F9;
+@gridNeutralBackgroundColor: #F8F8F9;
```

Comment:
Question: If it's called neutral, shouldn't this just use @neutralColor instead of a concrete value? Just asking for consistency in the naming, if it's not the same value than maybe rename it? WDYT?

Reply:
This is the light background color, which is brighter than the `@neutralColor` which is used on labels etc, so it shouldn't be the same, but maybe this one should be added to invenio-theme as well... ping @kpsherva

Reply:
Ah sorry, now I see what you mean. You're right, maybe it should rather be `@gridLightNeutralBackgroundColor` or just `@lightNeutralColor` or something similar.